external-crates/move/crates/move-compiler/tests/linter/true_positive_unnecessary_unit.exp

```diff
@@ -0,0 +1,143 @@
+warning[Lint W04010]: unit `()` expression can be removed or simplified
+  ┌─ tests/linter/true_positive_unnecessary_unit.move:6:16
+  │
+6 │         if (b) () else { x = 1 };
+  │             -  ^^ Unnecessary unit '()'
+  │             │   
+  │             Consider negating the 'if' condition and simplifying, e.g. 'if (cond) () else e' becomes 'if (!cond) e'
```

Comment:
Should this be a help message or a note on this diagnostic? I think if so it would make this a bit nicer.

Reply:
Maybe I'll split it it in half with part of it as a note